### ISSUE TITLE
TE Plaintext regression on 80-cores Ampere

### ISSUE THREAD

**Issue by EgorBo** [dotnet/runtime#97659]
![image](https://github.com/dotnet/runtime/assets/523221/670e12f8-dafb-422c-bd87-be71def54e29)

aspnet changes: https://github.com/dotnet/aspnetcore/compare/b43884c1b21e...481a5130567a
runtime changes: https://github.com/dotnet/runtime/compare/cf16ed8fa95f...b47fdeac709e

cc @sebastienros could it be also some of your experiments on that machine?

**Comment by sebastienros** [comment]
Definitely, I noticed it after I did a configuration change last week (which was suggested) and wanted to get a few runs before I revert it. I am reverting it now and expecting to see it revert. Interestingly some scenarios are not impacted in terms of throughput and still got latency improvements.

**Comment by EgorBo** [comment]
I see, closing it then since it's not a runtime issue.
